EasyDisenchant.lua

```diff
@@ -231,6 +238,42 @@ do
 		return button;
 	end
 
+	_M.IsItemInOutfitt = function(self, bagID, slotID, itemID)
+		if Outfitter then
+			local inventoryCache = Outfitter:GetInventoryCache()
+
+			-- Call this method to mark items with UsedInOutfit
+			inventoryCache:CompiledUnusedItemsList()
+
+			local vItems = inventoryCache.ItemsByCode[itemID]
+			if vItems ~= nil then
+				for _, vItemInfo in ipairs(vItems) do
+					local bagIndex = vItemInfo.Location.BagIndex
+					local bagSlotIndex = vItemInfo.Location.BagSlotIndex
+					
+					debug("Checking item: " .. vItemInfo.Link)
+					debug("bagIndex:", bagID, bagIndex)
+					debug("bagSlotIndex:", slotID, bagSlotIndex)
+					debug("itemID:", itemID, vItemInfo.Code)
+					debug("", vItemInfo)
+					
+					if bagIndex == bagID and bagSlotIndex == slotID and vItemInfo.Code == itemID then
+						debug("Check UsedInOutfit", vItemInfo.UsedInOutfit)
+
+						-- We found our item now check if its used in an outfitt.
```

Comment:
Typo in Outfit in comment and debug message.

Reply:
I will see what i can do